README.md

```diff
@@ -20,6 +20,23 @@ The Troup client is the frontend app that helps the user address problems that w
 -   [Prisma][prisma]
 -   [Nexus Schema][nexus-schema]
 
+## `.env` Setup
+
+You will need to add two `.env` files into the project to get it up and running. The first `.env` file should be created on the root of the project and contains the following information:
+
+```
+NODE_ENV="development"
+APP_SECRET="jwtsecret123"
+NODE_PATH="./src"
+DATABASE_URL="postgresql://prisma:prisma@localhost:5432/postgres?schema=public"
+```
+
+The second `.env` file should be created in the `prisma` folder and contains the following information:
+
+```
+DATABASE_URL="postgresql://prisma:prisma@localhost:5432/postgres?schema=public"
+```
+
```

Comment:
I have added two `example.env` files in the repo. These can be used to create the actual `.env` files. We should update the content to reflect the same, and redirect to those files.

```suggestion
## Setting up environment variables

You will need to create two `.env` files in the root and the `prisma` folder. The instructions on setting those up are available within each example:
- [Root environment variables](https://github.com/troup-io/troup-server/blob/master/example.env)
- [Prisma environment variables](https://github.com/troup-io/troup-server/blob/master/prisma/example.env)
```

Reply:
Perfect, that makes sense! Thanks